src/layouts/Images.jsx

```diff
@@ -68,18 +69,14 @@ export default class Images extends Component {
 
   uploadImage = async (imageName, imageContent) => {
     try {
-      const { match } = this.props;
-      const { siteName } = match.params;
-      const params = {
-        imageName,
-        content: imageContent,
-      };
-
-      await axios.post(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/images`, params, {
-        withCredentials: true,
+      // toggle state so that image renaming modal appears
+      this.setState({
+        uploadedImage: {
```

Comment:
maybe `pendingImageUpload` instead of `uploadedImage`?

Reply:
Fixed in 3ea87ca